vividus-plugin-visual/src/main/java/org/vividus/visual/engine/VisualTestingEngine.java

```diff
@@ -82,7 +83,9 @@ public VisualCheckResult compareAgainst(VisualCheck visualCheck) throws IOExcept
         {
             baselineScreenshot = EMPTY_SCREENSHOT;
         }
-        ImageDiffer differ = new ImageDiffer().withDiffMarkupPolicy(new PointsMarkupPolicy().withDiffColor(DIFF_COLOR));
+        PointsMarkupPolicy pointsMarkupPolicy = new PointsMarkupPolicy();
+        pointsMarkupPolicy.setDiffSizeTrigger(calculateDiffSizeTrigger(baselineScreenshot, checkpoint));
+        ImageDiffer differ = new ImageDiffer().withDiffMarkupPolicy(pointsMarkupPolicy.withDiffColor(DIFF_COLOR));
         ImageDiff diff = differ.makeDiff(baselineScreenshot, checkpoint);
 
         comparisonResult.setPassed(!diff.hasDiff());
```

Comment:
does it make sense to also print the diff percentage on fail? just to understand what number we should choose to make a test passed

Reply:
since it's a property applied to all tests, I don't see much sense